src/components/Nav.js

```diff
@@ -0,0 +1,82 @@
+import "../styles/nav.css";
+import { useUrl } from "../context/useUrl";
+import { useCart } from "../context/cartContext";
```

Comment:
club all exports from context folder into an index.js file .
Then will look like this ` import { useUrl , useCart } from "../context"`

Reply:
Thanks, now it looks clean I guess.